packages/ibm-products/src/components/Datagrid/useNestedRows.js

```diff
@@ -15,6 +15,24 @@ const blockClass = `${pkg.prefix}--datagrid`;
 const useNestedRows = (hooks) => {
   useNestedRowExpander(hooks);
   const useInstance = (instance) => {
+    useEffect(() => {
+      const { rows } = instance;
+      const defaultExpandedRows = rows.filter(
+        (row) => row.original.defaultExpanded
```

Comment:
```suggestion
        (row) => row?.original?.defaultExpanded
```

do these need to be optional?

Reply:
Yes, good catch. Safer to add the optional chaining